runtime/manifest-parser.peg

```diff
@@ -156,6 +176,16 @@ RecipeView
     }
   }
 
+RecipeSlot
+  = "renders" ref:(whiteSpace ViewRef)? name:(whiteSpace Name)? eolWhiteSpace
```

Comment:
ohk

advice needed, when i have:
RecipeSlot
  = "renders" ref:(whiteSpace lowerIdent)? name:(whiteSpace Name)? eolWhiteSpace

and the line is

renders as slot0

it parses "as" as the lowerIdent, and then expects to "as X" for the name.
what's the better way to formulate this?